### PR TITLE
Production mode

### LINKED ISSUE #92: Production mode changes
Most of the changes in this AAEC pull request will be needed for the app portfolio to run in production mode with MySQL.

https://github.com/uclibs/aaec/pull/70/files

### PR DESCRIPTION
**Pull request by Uzval-Mallepeddi**
Fixes #92 
- Changed sqlite to mysql db.
- Mailer for software request.
- Customized 404 and 500 pages.

### CODE REVIEW

**Comment by Uzval-Mallepeddi** [comment]
@hortongn The changes that you suggested are committed. But i'm not sure about the spec failure for mailer. Locally the specs passed gracefully. Any thoughts ?

File: app/mailers/admin_mailer.rb [hunk 1]
```diff
@@ -3,10 +3,14 @@
...
-      to: 'mallepvl@mail.uc.edu',
-      subject: 'New Software Request'
-    )
+
+  def new_software_request_mail(id, name)
+    @id = id
+    @name = name
+    @template = 'new_software_request_mail.html.erb'
+    @users = User.where(roles: 'root_admin')
+    @users.each do |admin|
```
Comment (hortongn) [inline app/mailers/admin_mailer.rb:12]: @Uzval-Mallepeddi The mail command will never run if there are no admin users.  That's why your admin_mailer_spec is failing.  CircleCI starts with a blank test database with no admin users.  So you need to create an admin user in your spec and it should pass.

Comment (Uzval-Mallepeddi) [reply app/mailers/admin_mailer.rb:12]: Alright! Thanks Glen. That worked like a charm!

**Comment by Uzval-Mallepeddi** [comment]
@hortongn Let me know if these changes are ok. I'll squash all the commits so that you can merge.

**Comment by hortongn** [comment]
@Uzval-Mallepeddi The app starts up fine in production mode.  Nice job.  Go ahead and squash and I'll merge.

**Pull request closed without merging**